### ISSUE TITLE
restart instead of leaving?

### ISSUE THREAD

**Issue by puppy65po** [skyloose/TDS#1]
i think it would make more sense since its meant to be a autofarm

**Comment by skyloose** [comment]
The current implementation may cause performance issues due to console warnings. I am actively working as we speak on a rewrite version of the autofarm

**Comment by puppy65po** [comment]
alrght! take your time, i only really care that it works! i feel like this script is the only one out there!

**Comment by skyloose** [comment]
Based on the current console messages, it appears the issue originates from the game itself
Rejoining the game appears to be the best solution to the console spam issue

**Comment by puppy65po** [comment]
Alright, so you are saying that the militant farm is rejoining the game to mitigate console spam issues. When it goes back to the lobby what is the script doing in order to play again? I've just been using the sniper farm for overnight since that one does use the restart button, and I find is more productive.

**Comment by skyloose** [comment]
Indeed, the issue does not stem from the executor but rather from the game itself. The script continuously searches for the map within the lobby, thereby repeating the sequence
